### PR TITLE
added login-system blog

### PR DESCRIPTION
**Pull request by akulsr0**

### CODE REVIEW

File: node/login-system/login-system-using-node-express.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+Lets get started
+
+Create a folder on Desktop as 'login-system' and initialize the project in there.
+
+    cd Desktop
+    mkdir login-system
+    cd login-system
+    npm init
+
+Change the entry point as _server.js_ and author as _your name_
```
Comment (itaditya) [inline node/login-system/login-system-using-node-express.md]: It's a best practice to not use italic in professional blog posts. Maybe use bold instead.

File: node/login-system/login-system-using-node-express.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+        const authResult = await bcrypt.compare(password, user[0].password);
+        if (authResult) {
+            const token = jwt.sign({ id: user[0]._id }, 'secretsecret', {
+            expiresIn: '24h'
+            });
+            res.cookie('token', token);
+            res.render('dashboard', {
+            name: user[0].name,
+            email: user[0].email,
+            age: user[0].age
```
Comment (itaditya) [inline node/login-system/login-system-using-node-express.md]: format the code

**Comment by itaditya** [comment]
This is a superb post. Crisp and to the point. One suggestion would be to deploy the project on heroku or glitch so people can interact with it.

**Comment by iamshadmirza** [comment]
Hey @akulsr0 
Are you going to address the comments so that I can merge the PR?
Let me know, please.

**Comment by akulsr0** [comment]
Hey @iamshadmirza, I would surely do the formatting stuff but for deploying I have to setup an online database, so maybe won't do it. Sounds good?

**Comment by iamshadmirza** [comment]
> Hey @iamshadmirza, I would surely do the formatting stuff but for deploying I have to setup an online database, so maybe won't do it. Sounds good?

Totally.
Let me when you're done.
Do join the telegram group for frequent updates. Would love to see more articles from you.

**Comment by akulsr0** [comment]
> > Hey @iamshadmirza, I would surely do the formatting stuff but for deploying I have to setup an online database, so maybe won't do it. Sounds good?
> 
> Totally.
> Let me when you're done.
> Do join the telegram group for frequent updates. Would love to see more articles from you.

Done! 
- Removed Italics
- Formatted Code

**Comment by iamshadmirza** [comment]
> > > Hey @iamshadmirza, I would surely do the formatting stuff but for deploying I have to setup an online database, so maybe won't do it. Sounds good?
> > 
> > Totally.
> > Let me when you're done.
> > Do join the telegram group for frequent updates. Would love to see more articles from you.
> 
> Done! 
> - Removed Italics
> - Formatted Code

Merged
You can publish this on hashnode or your personal blog
Then submit to the publication on hashnode with your original post link. 
It will be published
Let me know if you need any help.

**Pull request merged**